docs/content.zh/docs/dev/python/table/udfs/python_udfs.md

```diff
@@ -554,3 +554,11 @@ class ListViewConcatTableAggregateFunction(TableAggregateFunction):
     def get_result_type(self):
         return DataTypes.ROW([DataTypes.FIELD("a", DataTypes.STRING())])
 ```
+
+## 打包 UDFs
+
+To run Python UDFs (as well as Pandas UDFs) in any non-local mode, it is strongly recommended to bundle your Python UDF definitions using the config option [`python-files`]({{< ref "docs/dev/python/python_config" >}}#python-files), if your Python UDFs live outside of the file where the `main()` function is defined.
```

Comment:
It's not necessary to keep the English version if you have already translated it to Chinese. The paragraph should be removed.